lib/trino-hive-formats/src/test/java/io/trino/hive/formats/avro/TestAvroBase.java

```diff
@@ -326,6 +330,7 @@ private void testSerdeCycles(Schema schema, AvroCompressionKind compressionKind)
             }
         }
 
+        CodecFactory.addCodec(compressionKind.toString(), getLegacyCodecFactory(compressionKind));
```

Comment:
Why do this here?

Reply:
The spirit of the test is to use completely legacy Avro Code paths in a double serde loop with our page readers. To show full compatibility. To have the reader use the legacy codec factory the only way to pass it in is through the static map.